internal/config/config.go

```diff
@@ -6,11 +6,13 @@ import (
 
 type Config struct {
 	JWTTokenSecret string
+	Port           string
 }
 
 func NewConfig() Config {
 	config := Config{}
 
 	config.JWTTokenSecret = os.Getenv("JWTTokenSecret")
```

Comment:
names of env variables should be all capital with underscore delimiter. like `JWT_TOKEN_SECRET`.